components/counter-bar/src/stories.js

```diff
@@ -23,8 +24,7 @@ stories.add('Component default', () =>
       <CounterBar.Counter score={4}>Counter 4</CounterBar.Counter>
       <CounterBar.Counter score={5}>Counter 5</CounterBar.Counter>
     </CounterBar.Counters>
-  </CounterBar>),
-);
+  </CounterBar>));
```

Comment:
i'm really not keen on having the closing bracket here for the `stories.add` call at a different indent level - it really grates for me when i'm reading this code, and some editors may try to maintain the 4 spaces when writing the next line of code

i.e. the closing bracket should go onto the next line, as it was before

indeed the old code here was fine, except it should not have had the trailing `,`

this comment applies to the rest of this file too